cmd/openshift-install/create.go

```diff
@@ -111,14 +112,16 @@ var (
 				}
 
 				timer.StartTimer("Bootstrap Complete")
-				err = waitForBootstrapComplete(ctx, config)
+				errInfo := waitForBootstrapComplete(ctx, config)
+				err = errInfo.GetError()
 				if err != nil {
```

Comment:
You are missing a nil check here before calling `GetError`. But really, you don't need to store `errInfo` separately.
```suggestion
				if err := waitForBootstrapComplete(ctx, config); err != nil {
```